### DISCUSSION TITLE
package.json not updated, only package-lock.json

### DISCUSSION THREAD

**Discussion by markuspalme** [renovatebot/renovate#29776] [Request Help] [answered]
### What would you like help with?

I think I found a bug

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Azure DevOps, renovate@37.413.2

### Please tell us more about your question or problem

Renovate created a PR upgrading Angular from `18.0.2` to `18.0.3`. It only updated the file `package-lock.json´, not `package.json` resulting in a discrepancy in versions:

<img width="379" alt="image" src="https://github.com/renovatebot/renovate/assets/1481801/4d52da42-f959-4986-bda9-a6e6bdfb3e50">

Updated `package-lock.json`:

<img width="994" alt="image" src="https://github.com/renovatebot/renovate/assets/1481801/a2e6a265-4f24-4ed8-bb59-0fe181d37824">

`package.json`:

<img width="353" alt="image" src="https://github.com/renovatebot/renovate/assets/1481801/dfe06150-b7fc-42d4-82c6-e3aa6e2bb585">

My configuration is very basic.

Any idea on how to make Renovate update both files in sync?

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
This is not a bug - it's the correct default behavior.

Please read the docs on rangeStrategy

**Reply by markuspalme** [reply]
thank you!
